src/test/kotlin/com/ampnet/identityservice/controller/UserControllerTest.kt

```diff
@@ -62,7 +62,8 @@ class UserControllerTest : ControllerTestBase() {
             assertThat(user?.email).isEqualTo(testContext.email)
         }
         verify("Email verification token is not created") {
-            val mailToken = mailTokenRepository.findByUserAddress(testContext.user.address)
+            val mailToken = mailTokenRepository.findByUserAddressOrderByCreatedAtDesc(testContext.user.address)
+                .firstOrNull()
             assertThat(mailToken).isNull()
```

Comment:
Why get null and verify null?
```suggestion
            val mailTokens = mailTokenRepository.findByUserAddressOrderByCreatedAtDesc(testContext.user.address)
            assertThat(mailTokens).isEmpty()
```

Reply:
It is firstOrNull(), it's the same as verifying whether is it empty.